stockflux-securities-master/src/components/InputForm.js

```diff
@@ -114,7 +114,7 @@ const InputForm = ({ match }) => {
           <Components.LargeSpinner />
         </div>
       ) : (
-        <form className="input-form-body" onSubmit={submitForm}>
+        <div className="input-form-body">
```

Comment:
Why has the form submit handler been removed in favour of a click handler on the button? This stops us submitting the form by hitting return etc.

Reply:
Oh okay, I didn't realise that. I removed it because I was adding a second button which I didn't want to submit a form but I can add the form back and just add `type="button"` to the second button